example/mapboxExample.js

```diff
@@ -0,0 +1,145 @@
+import * as THREE from 'three';
+import CameraSync, { projectToWorld, projectedUnitsPerMeter } from './mapboxExampleCamera.js';
+import { DebugTilesRenderer as TilesRenderer } from '../src/index.js';
+
+function rotationBetweenDirections( dir1, dir2 ) {
+
+	const rotation = new THREE.Quaternion();
+	const a = new THREE.Vector3().crossVectors( dir1, dir2 );
+	rotation.x = a.x;
+	rotation.y = a.y;
+	rotation.z = a.z;
+	rotation.w = 1 + dir1.clone().dot( dir2 );
+	rotation.normalize();
+
+	return rotation;
+
+}
+
+const origin = [ 116, 35 ];
+
+// TO MAKE THE MAP APPEAR YOU MUST
+// ADD YOUR ACCESS TOKEN FROM
+// https://account.mapbox.com
+// This is my private key, it is prohibited to use in the project
+window.mapboxgl.accessToken =
+        'pk.eyJ1IjoieGlheGlhbmdmbmVnIiwiYSI6ImNscTRzc245NTA5c3cya3BhdXA4MDY0NWQifQ._9T_J9z7Hvcd00nQL3UuSw';
+const map = new window.mapboxgl.Map( {
+	container: 'map',
+	style: 'mapbox://styles/mapbox/dark-v9',
+	zoom: 17.86614600777933,
+	pitch: 70,
+	bearing: - 40,
+	center: origin,
+} );
+
+map.on( 'load', () => {
+
+	let renderer, scene, camera, world, tiles, tilesGroup;
+
+	map.addLayer( {
+		id: 'custom_layer',
+		type: 'custom',
+		onAdd: function ( map, mbxContext ) {
+
+			renderer = new THREE.WebGLRenderer( {
+				alpha: true,
+				antialias: true,
+				canvas: map.getCanvas(),
+				context: mbxContext,
+			} );
+
+			renderer.shadowMap.enabled = true;
+			renderer.autoClear = false;
+
+			scene = new THREE.Scene();
+			camera = new THREE.Camera();
+
+			world = new THREE.Group();
+			scene.add( world );
+
+			// lights
+			const dirLight = new THREE.DirectionalLight( 0xffffff, 0.8 );
+			dirLight.position.set( 1, 2, 3 );
+			world.add( dirLight );
+
+			const ambLight = new THREE.AmbientLight( 0xffffff, 1 );
+			world.add( ambLight );
+
+			tilesGroup = new THREE.Group();
+
+			world.add( tilesGroup );
+
+			new CameraSync( map, camera, world );
+
+			const url = 'https://xiaxiangfeng.github.io/3DTilesRendererJS.Test/baowei21/tileset.json';
+
+			tiles = new TilesRenderer( url );
```

Comment:
This is the tileset I generated using the tool, and all the b3DM files are placed in it https://xiaxiangfeng.github.io/3DTilesRendererJS.Test/ Under the directory, it can be downloaded

Reply:
Alternatively, you can modify it to another tileset

Reply:
What is the original source of the model, though? I would like to confirm the license and data used for the model.

Reply:
https://raw.githubusercontent.com/NASA-AMMOS/3DTilesSampleData/master/msl-dingo-gap/0528_0260184_to_s64o256_colorize/0528_0260184_to_s64o256_colorize/0528_0260184_to_s64o256_colorize_tileset.json